engines/director/score.cpp

```diff
@@ -623,6 +629,23 @@ void Score::renderFrame(uint16 frameId, bool forceUpdate, bool updateStageOnly)
 	}
 
 	g_system->copyRectToScreen(_surface->getPixels(), _surface->pitch, 0, 0, _surface->getBounds().width(), _surface->getBounds().height());
+
+}
+
+void Score::screenShot(){
+	Graphics::Surface rawSurface = _surface->rawSurface();
+	const Graphics::PixelFormat requiredFormat_4byte(4, 8, 8, 8, 8, 0, 8, 16, 24);
+	Graphics::Surface newSurface = *(rawSurface.convertTo(requiredFormat_4byte, _vm->getPalette()));
+
+	Common::String filename;
+	Common::String currentTarget = ConfMan.getActiveDomainName();
+	filename = Common::String::format("./dumps/%s-%05d.png",
+									  currentTarget.c_str(), _framesRan);
```

Comment:
Nope, this will not work with the current approach: you're running same target (director) hundreds of times for Workshop movies.

You'd rather use `dumpScriptName(archName.c_str(), type, id, "png")`